app/mailers/school_mailer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class SchoolMailer < ApplicationMailer
+  NOMINATION_EMAIL_TEMPLATE = "a7cc4d19-c0cb-4187-a71b-1b1ea029924f"
+
+  def nomination_email(recipient_email, nomination_url, reference)
+    template_mail(
+      NOMINATION_EMAIL_TEMPLATE,
+      to: recipient_email,
+      reference: reference,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        nomination_link: nomination_url,
```

Comment:
We need to include the school name here, see https://ukgovernmentdfe.slack.com/archives/C01J1J4FM50/p1616759041050100